### DISCUSSION TITLE
Modify status-radio.sh to restore RP metadata display

### DISCUSSION THREAD

**Discussion by markblinkhorn** [rern/rAudio#1136] [Tips] [answered]
Sometime after release 20221117 the RadioParadise metadata for the World Mix channel ceased to be displayed on the Player Status page. I did a bit of hacking around and discovered that the RP URL for the World Mix channel had changed from: 'https://stream.radioparadise.com/world-etc-flac' to 'https://stream.radioparadise.com/global-flac'  and thus was being missed by the case statement in /srv/http/bash/status-radio.sh

A small modification to the case statement restores the metadata display:

case $id in
        flac )   id=0;;
        mellow ) id=1;;
        rock )   id=2;;
        world )  id=3;;
        **global )  id=3;;**
        fip )           id=7;;
        fipelectro )    id=74;;
        fipgroove )     id=66;;
        fipjazz )       id=65;;
        fipnouveautes ) id=70;;
        fippop )        id=78;;
        fipreggae )     id=71;;
        fiprock )       id=64;;
        fipworld )      id=69;;
        francemusique )       id=4;;
        baroque )             id=408;;
        classiqueplus )       id=402;;
        concertsradiofrance ) id=403;;
        easyclassique )       id=401;;
        labo )                id=407;;
        lacontemporaine )     id=406;;
        lajazz )              id=405;;
        ocoramonde )          id=404;;
        opera )               id=409;;
esac

**Comment by markblinkhorn** [comment]
or you could just add another webradio station with the URL https://stream.radioparadise.com/world-etc-flac

**Comment by rern** [comment, marked as answer]
Thank you for the info. To be fixed by next update.
